docs/user/openstack/deploy_baremetal_workers.md

```diff
@@ -0,0 +1,96 @@
+- [Conciderations whe deploying bare-metal workers](#conciderations-whe-deploying-bare-metal-workers)
+- [Deploying cluster with BM workers on tenant network deployed by the installer](#deploying-cluster-with-bm-workers-on-tenant-network-deployed-by-the-installer)
+- [Deploying cluster with BM workers on preexisting network](#deploying-cluster-with-bm-workers-on-preexisting-network)
+- [Deploy Cluster with vms and  BM workers on preexisting network](#deploy-cluster-with-vms-and--bm-workers-on-preexisting-network)
+- [Deploy cluster with vm workers only and add BM workers after](#deploy-cluster-with-vm-workers-only-and-add-bm-workers-after)
+
+
+## Prerequisites
+* [Ironic bare metal service][2] is enabled an accessible through the [OpenStack Compute API][3].
+* Bare-metal workers are available as an [OpenStack flavor][1]
+* The image used to boot the bare-metal worker is available from the [Glance OpenStack image service][4] 
+* The cloud provider is configured such that bare-metal and vms can be attached to the same tenant network.
+* The cloud provider is configured such that vms can be deployed onto a preconfigured network which already has bare-metal servers attached to it.
+* [Ironic bare metal service][2] can listen for and PXE-boot nodes in tenant networks
+
+## Conciderations whe deploying bare-metal workers
+
+* Long boot times for bare-metal machines may result in deployment timeout
+
+    The first time a bare-metal server is booted the boot time could be significantly longer that
+    the first time a vm server is booted. This longer boot time could exceed the timeout settings of 
+    the installer. If that occurs, the deployment will fail with a timeout error.  The deployment maybe
+    restarted and completed by re-running  the installer with the appropriate *wait-for*
+    command. For example:
+        
+        ./openshift-install wait-for bootstrap-complete --log-level debug 
+          
+
+## Deploying cluster with BM workers on tenant network deployed by the installer
+The initial cluster is deployed using bare-metal workers only. Bare-metal workers and control-plane vms are all
+attached to the tenant network provisioned by the installer.
+
+- Create a install-config.yaml using the bare-metal [OpenStack flavor][1] as the *type* under the platform section of
+the worker machinepool section. Set the flavor for the masters vms under controlPLane.platform.type, and under
+the computeFlavor of platform.openstack. 
+
+        controlPlane:
+          platform:
+            openstack:
+              type: <vmComputeFlavorForMasters>
+        
+        ... other settings
+        compute:
+        - architecture: amd64
+          hyperthreading: Enabled
+          name: worker
+          platform:
+            openstack:
+              type: ocp-compute
+          replicas: 3
+
+        ... other settings
+        
+        platform:
+          openstack:
+            computeFlavor: <vmComputeFlavorForMasters>
```

Comment:
Tested leaving this empy with only controlPlane.platform.opesntack.type set and installer fails with 

FATAL failed to fetch Metadata: failed to load asset "Install Config": Unable to find flavor with name

Reply:
It fails at create ignition-configs:
[cloud-user@adduarte-installer bm]$ ./openshift-install create ignition-configs 
FATAL failed to fetch Kubeconfig Admin Client: failed to load asset "Install Config": Unable to find flavor with name  
[cloud-user@adduarte-installer bm]$

Reply:
we should add explanation that it will not be used but it can't be removed.